PharoTour/PharoTour.pillar

```diff
@@ -368,12 +367,22 @@ When a new System Browser window first opens, all panes are empty but the
 leftmost one. This first pane lists all known packages, which contain groups of
 related classes.
 
+@@authorToDo TODO: BTC-2015-08-26 double-check this next bit
+
+!!!! Expand the "Kernal" package and select "Objects"
 When we select a package, it causes the second pane to show a list of all of the
-classes in the selected package. When a class is selected the remaining two
+classes in the selected package. 
+
+!!!! Select the "Object" class
+When a class is selected the remaining two
 panes will be filled with text. The third pane displays the protocols of the
 currently selected class. These are convenient groupings of related methods. If
 no protocol is selected you should see all methods in the fourth pane.
 
+@@authorToDo: TODO-END: BTC-2015-08-26
```

Comment:
Done.  Now to help keep clean history I squashed this minor fix per [1]
$ git rebase -i HEAD~2
$ git push -f
Any concerns with that workflow?

[1] https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/